legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/transform/fromPure/tests/testToSQLString.pure

```diff
@@ -113,6 +113,18 @@ function <<test.Test>> meta::relational::tests::functions::sqlstring::testToSQLS
    assertEquals('select "root".ID as "pk_0", "root".FIRSTNAME as "firstName", "root".LASTNAME as "lastName", "root".AGE as "age" from catalog.schema.personTable as "root"', $s);
 }
 
+function <<test.Test>> meta::relational::tests::functions::sqlstring::testToSQLStringDatabricksSchemaNameShouldContainCatalogName():Boolean[1]
```

Comment:
I don't think this test should be in testToSQLString.pure. It should be in one of the db specific pure files. (But I see we also have other db specific tests in this file.)

This comment is a reminder that we have to relocate the db specific tests.